sentry/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: sentry
 description: A Helm chart for Kubernetes
 type: application
-version: 22.1.1
+version: 22.2.0
```

Comment:
Remove that please

Reply:
Hi @Mokto ,
I have removed this change. Thanks a lot.